tests/subsys/bluetooth/mesh/light_ctrl/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ target_compile_options(app
   -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_REG_LUX_PROLONG=200
   -DCONFIG_BT_MESH_LIGHT_CTRL_SRV_REG_LUX_STANDBY=100
   -DCONFIG_BT_MESH_LIGHT_CTRL_AMB_LIGHT_LEVEL_TIMEOUT=20
+  -DCONFIG_BT_MESH_SENSOR_USE_SENSOR_VALUE=1
```

Comment:
No, this is the way it is done in tests. They don't pick up the default values so they need to be specified here. Like all the -DCONFIG_...s above. The samples will pick up the default value and won't need this. (This will eventually be removed once the default changes)